components/Queue.js

```diff
@@ -40,16 +47,28 @@ class Queue extends React.Component {
             </div>
           </div>
           {isUserCourseStaff && (
-            <Button
-              color="danger"
-              tag="div"
-              outline
-              size="sm"
-              className="ml-auto"
-              onClick={e => this.deleteQueue(e)}
-            >
-              Delete
-            </Button>
+            <div className="ml-auto">
+              <Button
+                color="danger"
+                tag="div"
+                outline
+                size="sm"
+                className="mr-1 ml-auto"
+                onClick={e => this.deleteQueue(e)}
+              >
+                Delete
+              </Button>
+              <Button
+                color="primary"
+                tag="div"
+                outline
+                size="sm"
+                className="mr-0 ml-1"
+                onClick={e => this.updateQueue(e)}
+              >
+                Modify
```

Comment:
Should probably be "Edit" for consistency with the wording we use for questions.

Reply:
Done